docs/standard-library/allocators.md

```diff
@@ -18,6 +18,35 @@ class vector
 
 The C++ Standard Library provides a default implementation for an allocator. In C++11 and later, the default allocator is updated to expose a smaller interface; the new allocator is called a *minimal allocator*. In particular, the minimal allocator's `construct()` member supports move semantics, which can greatly improve performance. In most cases, this default allocator should be sufficient. In C++11 all the Standard Library types and functions that take an allocator type parameter support the minimal allocator interface, including `std::function`, `shared_ptr, allocate_shared()`, and `basic_string`.  For more information on the default allocator, see [`allocator` Class](allocator-class.md).
 
+## Writing Your Own Allocator (C++20)
+Several Allocator member functions have been removed: **`address`**, **`max_size`**, **`construct`** and **`destroy`**. **`allocate(n)`** creates array space of type T, but does not construct the objects. Possible exceptions are not handled in the minimal example below:
+
+```cpp
+include <iostream>
+#include <memory>
+
+int main()
+{
+  // allocator for integer values
+  std::allocator<int> myAllocator;
+
+  // allocate space for five ints
+  int* arr = myAllocator.allocate( 5 );
+
+  // construct and assign ints to elements 0 and 3
+  arr[ 0 ] = 100;
```

Comment:
our style is to not have spaces around array index, and to put the * with the variable, e.g. int *arr.  And we might as well name it what it is - myArray, or something.